docs/about/release-notes.md

```diff
@@ -119,6 +138,7 @@ to support such customization.
   repository, rather than to the root of the repository (#975), configurable
   via the new [`edit_uri`](../user-guide/configuration.md#edit_uri) setting.
 
+
```

Comment:
This extra line appears to be generating this [markdown-lint error](https://travis-ci.org/mkdocs/mkdocs/jobs/147590121#L198).